src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
++---
+title: Inclusive Making and Hacking
+layout: default
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
```

Comment:
And why are these kinds of events useful for education? What kind of learning opportunities do they afford that perhaps traditional classroom teaching doesn't? And do the present an opportunity to be somehow more inclusive, personalized, or open? (again, one sentence is plenty, but I'm thinking this little introduction should try to distill a few of the key concepts here very briefly)